.github/workflows/autolabel_action.yaml

```diff
@@ -0,0 +1,57 @@
+name: Autolabel PRs
+
+on: pull_request
+
+jobs:
+  apply_one_t_label:
+    runs-on: ubuntu-latest
+    # Don't run on forks, where we wouldn't have permissions to add the label anyway.
+    if: github.repository == 'leanprover-community/mathlib4'
+    permissions:
+      issues: write
+      checks: write
+      pull-requests: write
+      contents: read
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: install elan
+        run: |
+          set -o pipefail
+          curl -sSfL https://github.com/leanprover/elan/releases/download/v3.1.1/elan-x86_64-unknown-linux-gnu.tar.gz | tar xz
+          ./elan-init -y --default-toolchain none
+          echo "$HOME/.elan/bin" >> "${GITHUB_PATH}"
+
+      - name: Auto-label script
+        run: |
+          # the checkout dance, to avoid a detached head
+          git checkout master
+          git checkout -
+
+          # append the "noisy" commands, so we can read off their values and not fail CI
+          printf '\nproduce_labels! "git"\n\nproduce_labels "git"\n' >> Mathlib/Tactic/Autolabels/Labels.lean
+
+          # a first run to store the values in the CI logs -- useful for debugging
+          lake build Mathlib.Tactic.Autolabels.Labels
```

Comment:
Could you use `tee` to avoid running twice?

Reply:
I used `tee`, but I honestly find it less clear.

Reply:
What's less clear about it? Looks good to me.

Reply:
I'm not a big fan of these redirections, but I agree that this is almost as straightforward as it gets! 😊